crates/pop-contracts/src/utils/contracts_node.rs

```diff
@@ -39,36 +36,66 @@ pub async fn is_chain_alive(url: url::Url) -> Result<bool, Error> {
 	}
 }
 
+/// Checks if the `substrate-contracts-node` binary exists
+/// or if the binary exists in pop's cache.
+/// returns:
+/// - Some("", <version-output>) if the standalone binary exists
+/// - Some(binary_cache_location, "") if the binary exists in pop's cache
+/// - None if the binary does not exist
+pub fn does_contracts_node_exist(cache: PathBuf) -> Option<(PathBuf, String)> {
```

Comment:
As above, why 'utils' and not just a 'node' module in the root of the crate? A node is integral to using/testing/deploying contracts and should be a first-class citizen, not stuffed away under utils.